src/connected/connections.ts

```diff
@@ -88,7 +88,7 @@ export type ConnectionsNode = Connection | ConnectionGroup | RemoteProject | Wor
 export class AllConnectionsTreeDataProvider implements VSCode.TreeDataProvider<ConnectionsNode> {
   private readonly _onDidChangeTreeData = new VSCode.EventEmitter<Connection | undefined>();
   readonly onDidChangeTreeData: VSCode.Event<ConnectionsNode | undefined> = this._onDidChangeTreeData.event;
-  private allConnections = { sonarqube: Array.from<Connection>([]), sonarcloud: Array.from<Connection>([]) };
+  private allConnections = { __sonarqube__: Array.from<Connection>([]), __sonarcloud__: Array.from<Connection>([]) };
```

Comment:
This was a bug that was preventing the connection check results to be reflected in the connections list (i.e. change from loading to successful status)